### PR TITLE
Adds a NT Baseball uniform to counter the Syndicate one, and adds associated hats

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
Title
I've had this laying around since the syndicate icon pr, so I figured I'd toss it in-game later rather than never.

Not currently available to anyone outside admin intervention, BUT hopefully it leads to some silly gimmicks and/or events.

## How This Contributes To The Skyrat Roleplay Experience
image
We ballin'

## Changelog

:cl:
add: Added an NT Baseball Uniform, parallel the Syndicate one. Also added hats to pair with both. (Not crew-obtainable, but feel free to make a gimmick from them!)
/:cl:

### CODE REVIEW

**Comment by theselfish** [comment]
Are you sure upstream wouldn't like this?

EDIT: Wait. The baseball stuff is ours, isn't it?

**Comment by ghost** [comment]
> Are you sure upstream wouldn't like this?
> 
> EDIT: Wait. The baseball stuff is ours, isn't it?

TBH they MIGHT. The uniforms are ours, the pitch, balls, bats, etc are TG's
I just have no clue how I'd put them up there, where, how, etc

File: modular_skyrat/master_files/code/modules/clothing/under/jobs/command.dm [hunk 1]
```diff
@@ -145,6 +145,11 @@
 *	MISC
 */
 
+/obj/item/clothing/under/rank/captain/skyrat/baseball
+	name = "nanotrasen baseball tee"
+	desc = "Up to throw we have the NanoTrasen Thuggers, already winding up a plasma-fuelled pitch!"
```
Comment (tf-4) [inline modular_skyrat/master_files/code/modules/clothing/under/jobs/command.dm]:
```suggestion
	desc = "Up to throw we have the Nanotrasen Thuggers, already winding up a plasma-fuelled pitch!"
```

File: modular_skyrat/master_files/code/modules/clothing/under/jobs/command.dm [hunk 2]
```diff
@@ -145,6 +145,11 @@
 *	MISC
 */
 
+/obj/item/clothing/under/rank/captain/skyrat/baseball
```
Comment (tf-4) [inline modular_skyrat/master_files/code/modules/clothing/under/jobs/command.dm:148]: Why is it a captain subtype exactly?

Comment (ghost) [reply modular_skyrat/master_files/code/modules/clothing/under/jobs/command.dm:148]: All generic command gear is 🤷 God bless SS13

I'll probably toss this upstream anyways so it'll most likely end up moving to a greyscale costume. Gonna leave this PR up for da reviews tho 😏

**Comment by Iamgoofball** [comment]
the baseball stuff is upstream, and i think we'd accept baseball outfits upstream

obligatory:
177752120-92a2440f-4547-491c-b2e7-a8757a57f6b6

**Review by Gandalf2k15 — CHANGES_REQUESTED** [review]
ye ye ye take er upstream

**Pull request closed without merging**